SUMMARY.md

```diff
@@ -4,43 +4,30 @@
-    - [Design Verification](./hw/top_earlgrey/ip_autogen/rv_plic/doc/dv/README.md)
-      - [Testplan](./hw/top_earlgrey/ip_autogen/rv_plic/data/rv_plic_fpv_testplan.hjson)
-    - [Programmer's Guide](./hw/top_earlgrey/ip_autogen/rv_plic/doc/programmers_guide.md)
-    - [Interface and Registers](./hw/top_earlgrey/ip_autogen/rv_plic/data/rv_plic.hjson)
-    - [Device Interface Functions](./sw/device/lib/dif/dif_rv_plic.h)
-    - [Checklist](./hw/top_earlgrey/ip_autogen/rv_plic/doc/checklist.md)
-  - [Sensor Control](./hw/top_earlgrey/ip/sensor_ctrl/README.md)
-    - [Theory of Operation](./hw/top_earlgrey/ip/sensor_ctrl/doc/theory_of_operation.md)
-    - [Programmer's Guide](./hw/top_earlgrey/ip/sensor_ctrl/doc/programmers_guide.md)
-    - [Interface and Registers](./hw/top_earlgrey/ip/sensor_ctrl/data/sensor_ctrl.hjson)
-    - [Device Interface Functions](./sw/device/lib/dif/dif_sensor_ctrl.h)
-    - [Checklist](./hw/top_earlgrey/ip/sensor_ctrl/doc/checklist.md)
-  - [TL-UL Checklist](./hw/top_earlgrey/ip/xbar/doc/checklist.md)
-  - [Pinmux Targets](./hw/top_earlgrey/ip/pinmux/doc/autogen/targets.md)
-    - [ASIC Target Pinout and Pinmux Connectivity](./hw/top_earlgrey/ip/pinmux/doc/autogen/pinout_asic.md)
-    - [CW310 Target Pinout and Pinmux Connectivity](./hw/top_earlgrey/ip/pinmux/doc/autogen/pinout_cw310.md)
+- [HW Homepage](./hw/README.md)
+
+- [Tops](./doc/tops.md)
+  - [Earl Grey](./hw/top_earlgrey/README.md)
+    - [Datasheet](./hw/top_earlgrey/doc/datasheet.md)
+    - [Design Specification](./hw/top_earlgrey/doc/design/README.md)
+    - [Design Verification](./hw/top_earlgrey/dv/README.md)
+      - [Chip Testplan](./hw/top_earlgrey/data/chip_testplan.hjson)
+    - [Top Generation](./hw/top_earlgrey/doc/design/top_generation.md)
+    - [Earl Grey HWIP](./hw/top_earlgrey/doc/design/hwip.md)
+      - [Analog Sensor Top](./hw/top_earlgrey/ip/ast/README.md)
+        - [Interface Signals](./hw/top_earlgrey/ip/ast/doc/interfaces.md)
+      - [Sensor Control](./hw/top_earlgrey/ip/sensor_ctrl/README.md)
+        - [Theory of Operation](./hw/top_earlgrey/ip/sensor_ctrl/doc/theory_of_operation.md)
+        - [Programmer's Guide](./hw/top_earlgrey/ip/sensor_ctrl/doc/programmers_guide.md)
+        - [Interface and Registers](./hw/top_earlgrey/ip/sensor_ctrl/data/sensor_ctrl.hjson)
+        - [Device Interface Functions](./sw/device/lib/dif/dif_sensor_ctrl.h)
+        - [Checklist](./hw/top_earlgrey/ip/sensor_ctrl/doc/checklist.md)
+      - [Pinmux Targets](./hw/top_earlgrey/ip/pinmux/doc/autogen/targets.md)
+        - [ASIC Target Pinout and Pinmux Connectivity](./hw/top_earlgrey/ip/pinmux/doc/autogen/pinout_asic.md)
+        - [CW310 Target Pinout and Pinmux Connectivity](./hw/top_earlgrey/ip/pinmux/doc/autogen/pinout_cw310.md)
 
 - [Cores](./hw/doc/cores.md)
-  - [Ibex RISC-V Core Wrapper](./hw/ip/rv_core_ibex/README.md)
+  - [Ibex RISC-V Core](./hw/ip/rv_core_ibex/README.md)
```

Comment:
I think these docs are just for the wrapper and not for Ibex itself.

Should we link to the [Ibex documentation](https://ibex-core.readthedocs.io/en/latest/) here and move the wrapper's docs into the comportable IP section?

Reply:
Good idea, I wanted to remove 'Wrapper' from the title because it felt inelegant. Updated the link here (under 'Cores') to the ibex projects/docs, and moved the pages here down under hardware blocks and changed back to 'Wrapper'.